Examples/Run/Seeding/Common/SeedingExample.cpp

```diff
@@ -92,11 +95,37 @@ int runSeedingExample(int argc, char* argv[],
   sequencer.addAlgorithm(
       std::make_shared<HitSmearing>(hitSmearingCfg, logLevel));
 
+  // Create space points
+  SpacePointMaker::Config spCfg;
+  spCfg.inputSourceLinks = hitSmearingCfg.outputSourceLinks;
+  spCfg.inputMeasurements = hitSmearingCfg.outputMeasurements;
+  spCfg.outputSpacePoints = "spacepoints";
+  spCfg.trackingGeometry = tGeometry;
+  spCfg.geometrySelection = {
+      // barrel pixel layers
+      Acts::GeometryIdentifier().setVolume(8).setLayer(2),
+      Acts::GeometryIdentifier().setVolume(8).setLayer(4),
+      Acts::GeometryIdentifier().setVolume(8).setLayer(6),
+      // positive endcap pixel layers
+      Acts::GeometryIdentifier().setVolume(9).setLayer(2),
+      Acts::GeometryIdentifier().setVolume(9).setLayer(4),
+      Acts::GeometryIdentifier().setVolume(9).setLayer(6),
+      Acts::GeometryIdentifier().setVolume(9).setLayer(8),
+      // negative endcap pixel layers
+      Acts::GeometryIdentifier().setVolume(7).setLayer(14),
+      Acts::GeometryIdentifier().setVolume(7).setLayer(12),
+      Acts::GeometryIdentifier().setVolume(7).setLayer(10),
+      Acts::GeometryIdentifier().setVolume(7).setLayer(8),
+  };
+  sequencer.addAlgorithm(std::make_shared<SpacePointMaker>(spCfg, logLevel));
+
   // Seeding algorithm
   SeedingAlgorithm::Config seedingCfg;
+  seedingCfg.inputSpacePoints = {
+      spCfg.outputSpacePoints,
```

Comment:
```suggestion
      spCfg.outputSpacePoints
```

Reply:
This is intentional to clarify that this initializes a container with potentially more than one entry. It also makes `clang-format` put every entry on a separate line which I find easier to read.